pkgs/development/ruby-modules/with-packages/test.nix

```diff
@@ -0,0 +1,94 @@
+# a generic test suite for all gems for all ruby versions.
+# use via nix-build.
+let
+  pkgs = import ../../../.. {};
+  lib = pkgs.lib;
+  stdenv = pkgs.stdenv;
+
+  rubyVersions = with pkgs; [
+    # ruby_2_4
+    ruby_2_5
+    # ruby_2_6
+  ];
+
+  gemTests = (builtins.mapAttrs (name: gem:
+    ''
+      require "${name}"
+    ''
+  ) pkgs.ruby.gems) // {
+    actioncable = "require 'action_cable'";
+    actionmailer = "require 'action_mailer'";
+    actionpack = "require 'action_pack'";
+    actionview = "require 'action_view'";
+    activejob = "require 'active_job'";
+    activemodel = "require 'active_model'";
+    activerecord = "require 'active_record'";
+    activestorage = "require 'active_storage'";
+    activesupport = "require 'active_support'";
+    atk = "puts 'skip'";
```

Comment:
?

Reply:
Still trying to figure out why `propagatedBuildInputs` aren't being propagated... might be related to `buildEnv`.